core/changestream/subscription.go

```diff
@@ -47,9 +43,6 @@ func (o SubscriptionOption) Filter() func(ChangeEvent) bool {
 // Namespace returns a SubscriptionOption that will subscribe to the given
 // namespace.
 func Namespace(namespace string, changeMask ChangeType) SubscriptionOption {
-	if changeMask == 0 {
-		panic(fmt.Errorf("unexpected changeMask value: 0"))
-	}
```

Comment:
This isn't useful. If someone sends a 0 or any int that isn't within the ChangeType, then don't write a test, it's on them. We shouldn't special case 0. Ultimately now, they just won't get any events.